### DISCUSSION TITLE
gnutls_handshake() failed: The TLS connection was non-properly terminated.

### DISCUSSION THREAD

**Discussion by atvcaptain** [community/community#70832] [Repositories] [answered]
### Select Topic Area

General

### Body

start 16:00 Uhr german Time

cant fetch git refs i get

fatal: unable to access 'https://github.com/openatv/enigma2.git/': gnutls_handshake() failed: The TLS connection was non-properly terminated.

**Comment by jenseneverest** [comment]
Been driving me nuts for a few hours here in the UK

**Reply by jenseneverest** [reply, marked as answer]
seems to have been resolved for now
